plugins/inputs/varnish/varnish.go

```diff
@@ -157,6 +232,7 @@ func (s *Varnish) Gather(acc telegraf.Accumulator) error {
 	for section, fields := range sectionMap {
 		tags := map[string]string{
 			"section": section,
+			"vcl":     vclactive,
```

Comment:
what are your thoughts on adding it as a field named vcl instead? Due to the timestamp being included in the name of the vcl there are concerns around cardinality with it as a tag.

Reply:
You will have as many tags as reload you run. Maybe could make sense send this tag if user configure it, right?

Reply:
It could be a user configured tag option or if it's a single field it should avoid cardinality issues.